app/components/Form/ProjectManagerForm.tsx

```diff
@@ -87,7 +86,7 @@ const ProjectManagerForm: React.FC<Props> = (props) => {
       anyOf: allCifUsers.edges.map(({ node }) => {
         return {
           type: "number",
-          title: `${node.firstName} ${node.lastName}`,
+          title: `${node.fullName}`,
```

Comment:
```suggestion
          title: node.fullName,
```
no need for string interpolation here